packages/commons/tests/unit/LRUCache.test.ts

```diff
@@ -86,7 +82,7 @@ describe('Class: LRUMap', () => {
       expect(value).toBeUndefined();
     });
 
-    test('when called it marks the item as the most recently used', () => {
+    it('marks the item as the most recently used', () => {
```

Comment:
I'm not sure if this test is testing something that hasn't been tested before.
You may don't need this.

Reply:
You're right, good catch!

Removed

Reply:
Turns out the test was needed, if I remove it coverage drops and CI fails.